lib/util.ts

```diff
@@ -31,25 +37,25 @@ const PageLocation = parquet_thrift.PageLocation.prototype = [];
 PageLocation.write = previousPageLocation.write;
 PageLocation.read = previousPageLocation.read;
 
-const getterSetter = index => ({
-  get: function() { return this[index]; },
-  set: function(value) { return this[index] = value;}
+const getterSetter = (index: number) => ({
+  get: function(): number { return this[index]; },
+  set: function(value: number): number { return this[index] = value;}
 });
 
 Object.defineProperty(PageLocation,'offset', getterSetter(0));
 Object.defineProperty(PageLocation,'compressed_page_size', getterSetter(1));
 Object.defineProperty(PageLocation,'first_row_index', getterSetter(2));
 
 
-exports.force32 = function() {
+export const force32 = function() {
   const protocol = thrift.TCompactProtocol.prototype;
   protocol.zigzagToI64 = protocol.zigzagToI32;
   protocol.readVarint64 = protocol.readVarint32 = function() {
```

Comment:
There is no type definition for these files in the imported types module but it exists in the actual module